tests/cephfs/cephfs_nfs/test_all_nfs_vers_mount_and_perform_failover.py

```diff
@@ -0,0 +1,216 @@
+import random
+import re
+import secrets
+import string
+import time
+import traceback
+
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-83575085 - Create an export on the NFS cluster and mount it using all
+                    supported NFS versions(4,1,4.2) using the backend servers as well as the ingress
+                    and perform IO as usual.
+    Test operation:
+        1. Create a nfs cluster
+        2. Create a nfs export
+        3. Create 2 exports
+        4. Mount 1st export with nfs version 4.1 and run IOs
```

Comment:
Can we also include version as 4 in the same test? We might be covering it other test. Just a thought, open for discussion.

Reply:
added version 4 also.